filters/json.js

```diff
@@ -69,6 +69,6 @@ export class JsonFilter {
                 }
             }
         }
-        return flat.unflatten(flatResource);
+        return JSON.stringify(flat.unflatten(flatResource), null, 2);
```

Comment:
how to stringify is highly opinionated. for example, i prefer `\t` to save 1 char in output. since it sounds hard to detect what was used in input maybe it makes sense to make it configurable?